WordPress/Classes/ViewRelated/Views/BottomSheet/BottomSheetPresentationController.swift

```diff
@@ -0,0 +1,251 @@
+
+import UIKit
+
+// MARK: - BottomSheetPresentationControllerConstants
+
+private struct BottomSheetPresentationControllerConstants {
```

Comment:
Would it make sense to make this an internal type to `BottomSheetPresentationController`, considering it's mostly used as such? Also would that make the names a bit less verbose?